pkgs/development/python-modules/azure-mgmt-appcontainers/default.nix

```diff
@@ -5,6 +5,7 @@
 , azure-common
 , azure-mgmt-core
 , isodate
+, msrest
```

Comment:
I'm wondering where `msrest` is coming from as there is no mentioning in [`setup.py`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appcontainers/azure-mgmt-appcontainers/setup.py#L76-L80) for 3.0.0. Also, 3.0.0 seems to build on [Hydra](https://hydra.nixos.org/eval/1798815?filter=azure-mgmt-appcontainers&compare=1798809&full=).

I guess that the overrides in `pkgs/tools/admin/azure-cli/python-packages.nix` should be adjusted. E.g., `azure-mgmt-appcontainers-2.0.0` need `msrest`.

Reply:
Fixed by adding `msrest` to specific packages in the overrides instead of the global packages.